docs/source/07_extend_kedro/03_custom_datasets.md

```diff
@@ -255,16 +256,14 @@ class ImageDataSet(AbstractDataSet):
         Returns:
             Data from the image file as a numpy array
         """
-        # using get_filepath_str ensures that the protocol and path are appended correctly for different filesystems
-        load_path = get_filepath_str(self._get_load_path(), self._protocol)
+        load_path = get_filepath_str(self._filepath, self._protocol)
         with self._fs.open(load_path, mode="r") as f:
```

Comment:
Just realised, it'd be good to update these `open` calls as well - with newer version of `pylint` or `flake8` they make you explicitly use `encoding`, so we can add an `encoding="utf-8"` while we're here.

Reply:
Good call - will do!

Reply:
Actually looking through our codebase it looks like we don't put `encoding` on `self._fs.open` calls, just on `open` function calls?

Reply:
Doesn't matter then, thanks for checking.